examples/webpack.config.local.js

```diff
@@ -29,16 +30,15 @@ function makeLocalDevConfig(EXAMPLE_DIR = LIB_DIR) {
 
     resolve: {
       alias: {
-        // For importing modules that are not exported at root
         'nebula.gl/dist': CORE_SRC_DIR,
-        // Imports the nebula.gl library from the src directory in this repo
         'nebula.gl': CORE_SRC_DIR,
 
-        // For importing modules that are not exported at root
         'nebula.gl-react/dist': REACT_SRC_DIR,
-        // Imports the nebula.gl library from the src directory in this repo
         'nebula.gl-react': REACT_SRC_DIR,
 
+        '@nebula.gl/overlays/dist': OVERLAYS_SRC_DIR,
```

Comment:
* Why do you need a sepate alias to dist? In other repos, we switch the alias between src and dist to enable testing from source or compiled code.
* I guess there should be an alias for each module?
* We often have an `aliases.js` file at the root listing the aliases that we just import here and in other places.

Reply:
More help on how to do this?